lib/dlss/capistrano/tasks/control_master.rake

```diff
@@ -17,6 +17,17 @@ namespace :deploy do
   end
 end
 
+namespace :shared_configs do
+  before :check, :setup_connection do
+    invoke 'controlmaster:setup'
+    invoke 'otk:generate'
```

Comment:
without this, no connections worked for the `shared_configs` tasks, pre-existing control master or no